e2e/portal/tests/ProfilePageActivityLog/ViewActivityOverview.spec.ts

```diff
@@ -167,13 +170,75 @@ test.describe('as admin user', () => {
   });
 });
 
-test.describe('as user with fewer permissions', () => {
-  test.beforeEach(async ({ page }) => {
+test.describe('as user with only view paper voucher permissions', () => {
+  test('transaction row contains current balance, but not "view voucher" button', async ({
+    page,
+  }) => {
+    // Login as admin
+    await loginAs({
+      page,
+      username: env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    });
+    const personalInformationPage = new RegistrationPersonalInformationPage(
+      page,
+    );
+
+    // Navigate to the personal info page and change FSP to Albert Heijn voucher paper
+    const activityLogPage = new RegistrationActivityLogPage(page);
+
+    const tableComponent = new TableComponent(page);
+    const registrationsPage = new RegistrationsPage(page);
+
+    await activityLogPage.selectProgram(NLRCProgram.titlePortal.en);
+    await registrationsPage.goToRegistrationByName({
+      registrationName: registrationPV5.fullName,
+    });
+
+    await activityLogPage.navigateToPersonalInformation();
+
+    await personalInformationPage.clickEditInformationButton();
+
+    // Change FSP from dropdown selection and fill in all the required fields
+    await personalInformationPage.selectDropdownOption({
+      dropdownIdName: 'programFspConfigurationName',
+      dropdownLabel: dropdownInputs.programFspConfigurationName.fieldName,
+      option: 'Albert Heijn voucher paper',
+    });
+
+    await personalInformationPage.saveChanges();
+
+    const accessToken = await getAccessToken();
+
+    // Make a second payment with the new FSP
+    const paymentResponse = await createAndStartPayment({
+      programId: 2,
+      transferValue: 100,
+      referenceIds: [referenceIdPV5],
+      accessToken,
+    });
+    const paymentId = paymentResponse.body.id;
+
+    await waitForPaymentTransactionsToComplete({
+      programId: programIdPV,
+      paymentId,
+      paymentReferenceIds: [referenceIdPV5],
+      accessToken,
+      maxWaitTimeMs: 2_000,
+      completeStatuses: [
+        TransactionStatusEnum.success,
+        TransactionStatusEnum.waiting,
+      ],
+    });
```

Comment:
you can use doPaymentAndWaitForCompletion tot combine these 2 calls. (Same btw in general beforeEach of this file)